cmd/solana/main.go

```diff
@@ -46,7 +51,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("walletRepo error: %v", err)
 	}
-	walletService := wallet.NewService(repo, solana.New(client.DevnetRPCEndpoint))
+	walletService := wallet.NewService(repo, solana.New(client.DevnetRPCEndpoint), feePayerAccountPrivateKey, assetAccountPrivateKey, issuerAccountPrivateKey)
```

Comment:
what do you think about passing of fee payer, asset, and issuer directly to Solana wrapper?